integration-tests/README.md

```diff
@@ -13,8 +42,49 @@ Before running integration tests, ensure you have:
 - `DT_ENVIRONMENT` - Your Dynatrace environment URL (e.g., `https://abc123.apps.dynatrace.com`)
 - `DT_PLATFORM_TOKEN` - (Optional) Your Dynatrace platform token
 
-Run tests via
+**Required OAuth Scopes:**
```

Comment:
We don't need specific oauth scopes for integration tests. Ideally, we use exactly the same scopes as we defined in the main readme. Please remove those changes.